packages/react-select/src/Select.tsx

```diff
@@ -1992,7 +2009,7 @@ export default class Select<
               />
             ))
           ) : (
-            <input name={name} type="hidden" />
+            <input name={name} type="hidden" value="" />
```

Comment:
Just wondering, does setting value="" mean that it can't be used as an uncontrolled component anymore?

Reply:
It would prevent React from throwing a "Switching from uncontrolled to controlled" error because React reuses the `<input>` element for `RequiredInput` component and if a value is set.

I don't know if it was really applicable here anymore, but better safe than sorry.